modules/product/src/main/java/com/opengamma/strata/product/index/type/IborFutureTemplate.java

```diff
@@ -77,37 +36,54 @@
    * @return the template
    */
   public static IborFutureTemplate of(Period minimumPeriod, int sequenceNumber, IborFutureConvention convention) {
-    return IborFutureTemplate.builder()
-        .minimumPeriod(minimumPeriod)
-        .sequenceNumber(sequenceNumber)
-        .convention(convention)
-        .build();
+    return RelativeIborFutureTemplate.of(minimumPeriod, sequenceNumber, convention);
+  }
+
+  /**
+   * Obtains a template based on the specified convention using an absolute definition of time.
+   * <p>
+   * The future is selected from a sequence of futures based on a year-month.
+   * In most cases, the date of the future will be in the same month as the specified month,
+   * but this is not guaranteed.
+   * 
+   * @param yearMonth  the year-month to use to select the future
+   * @param convention  the future convention
+   * @return the template
+   */
+  public static AbsoluteIborFutureTemplate of(YearMonth yearMonth, IborFutureConvention convention) {
```

Comment:
I think the return type should be `IborFutureTemplate` as `AbsoluteIborFutureTemplate` is package-private.